mollie.php

```diff
@@ -1131,10 +1133,14 @@ public function hookActionEmailSendBefore($params)
         }
 
         if ($params['template'] === 'order_conf') {
-            if (Configuration::get(\Mollie\Config\Config::MOLLIE_SEND_ORDER_CONFIRMATION)) {
-                return true;
+            switch (Configuration::get(\Mollie\Config\Config::MOLLIE_SEND_ORDER_CONFIRMATION)) {
```

Comment:
Where should we move this logic? Can we create validator for this switch? Or should this be utility?

Reply:
Validator sounds good